tools/proofers/my_projects.php

```diff
@@ -297,30 +334,213 @@ function sql_order_spec( $order_col, $order_dir )
     echo "<br>\n";
 }
 
-// -----------------
-function get_sort_col_and_dir()
+// --------------------------------------------------------------------------
+
+function get_table_column_specs()
+{
+    // $colspecs = array (
+    //     $id => array ( 'label' => $label, 'sql' => $sql )
+    // );
+    // $id is the column name as passed by GET argument.
+    // $label is the translatable label displayed in the column header
+    // $sql is the SQL collater expression, or NULL for unsortable columns.
+    // $class is the HTML class to use for the field on output
+    $round_columns = [
+        'title' => [
+            'label' => _('Title'),
+            'sql'   => 'projects.nameofwork',
+        ],
+        'state' => [
+            'label' => _('Current State'),
+            'sql'   => sql_collater_for_project_state('projects.state'),
+        ],
+        'time' => [
+            'label' => _('Time of Last Activity'),
+            'sql'   => 'max_timestamp',
+        ],
+        'n_available_pages' => [
+            'label' => _('Available<br>Pages'),
+            'sql'   => 'n_available_pages',
+            'class' => 'right-align',
+        ],
+        'percent_done' => [
+            'label' => _('Done'),
+            'sql'   => 'percent_done',
+            'class' => 'right-align',
+        ],
+        'days_checkedout' => [
+            'label' => _('Days'),
```

Comment:
For the `Recent`, `Available`, and `Active` tabs in the `Proofreading & Formatting` section, perhaps just label it "Days in State"? The `Active` tab for `Post-Processing Projects` already says "Days Checked Out".

Reply:
I've changed it to `Days in State` in my sandbox. It's clearer what it is, but it makes the column header taller than I'd like. Not sure there's anything to do about it however.

Reply:
Whether the column heading shows as two or three lines depends on the width of the window. `Time of Last Activity` is a long heading, too, but that column can't be forced to be narrow enough to push it to 3 lines. `Available Pages` can't be pushed narrower because the column heading is only 2 words. Would it be frowned upon to put a non-breaking between `Days` and `in`? If so, I think we just have to live with it. (But then there are other tables with the same problem -- at a quick glance, the personal PP and PPV tables at the top of the PP and PPV pool pages.)

Reply:
I think the possibility of it going to 3 lines is worth the clarity.